packages/info/logger/mixin.core.js

```diff
@@ -14,6 +14,19 @@ class LogMixin extends Mixin {
     this.handleArguments(this.options);
   }
 
+  configureServer(app, middlewares, mode) {
+    if (mode === 'static') {
+      return;
+    }
+
+    middlewares.preinitial.unshift((_, res, next) => {
+      Object.assign(res.locals, {
+        logger: this.logger,
```

Comment:
I think we need to create a new logger for each request.

Reply:
In the context of this home-grown logger of ours? Or in the context of sending request-specific logs to Logjam? Because the latter requires additional plumbing anyways.